server_action_navigate/models/base.py

```diff
@@ -0,0 +1,62 @@
+# Migrated to v14.0 by Ashish Hirpara (https://www.ashish-hirpara.com)
```

Comment:
please remove this line, it's not your copyright here

Reply:
also, I fail to find where this is coming from... any pointer?

Reply:
this function was in odoo core and was removed after 13.0 see : 

- https://github.com/odoo/odoo/blob/13.0/odoo/models.py#L4892

However, I don't know what's the best design, but copying the whole function in this module doesn't seems to be a good idea.
- maybe a dedicated module
- or remove the call, and write a work around.

@AshishHirapara, what do you think ? Do you need some help on that topic ? 

regards.

Reply:
> this function was in odoo core and was removed after 13.0 see :
> 
> * [https://github.com/odoo/odoo/blob/13.0/odoo/models.py#L4892](https://github.com/odoo/odoo/blob/13.0/odoo/models.py?rgh-link-date=2022-04-20T08%3A56%3A42Z#L4892)
> 
> However, I don't know what's the best design, but copying the whole function in this module doesn't seems to be a good idea.
> 
> * maybe a dedicated module
> * or remove the call, and write a work around.
> 
> @AshishHirapara, what do you think ? Do you need some help on that topic ?
> 
> regards.


Thanks, @legalsylvain. I got that the function is deleted from 14, and if you think it's a bad idea, then help me here.
You have suggested 2 solutions here.
1. A dedicated module: what do you mean by a dedicated module? Just for the function? Because it will be ultimately the same, right?
2. Remove the call: I can remove the function call and write the whole function code at the location. Will that be a proper solution?

Thanks